src/main/java/com/exactprosystems/remotehand/web/actions/TakeElementScreenshotSnapshot.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2020-2021 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactprosystems.remotehand.web.actions;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+import com.exactprosystems.remotehand.utils.ScreenshotRegionUtils;
+import com.exactprosystems.remotehand.web.WebConfiguration;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.exactprosystems.remotehand.ScriptCompileException;
+import com.exactprosystems.remotehand.ScriptExecuteException;
+import com.exactprosystems.remotehand.web.WebAction;
+
+public class TakeElementScreenshotSnapshot extends WebAction
```

Comment:
`TakeElementScreenshotSnapshot ` - this name confuses me. Maybe, call this action `SaveElementScreenshot ` or `StoreElementScreenshot`?